lib/vtls/openssl.c

```diff
@@ -2719,6 +2725,157 @@ static CURLcode ossl_connect_step1(struct connectdata *conn, int sockindex)
   }
 #endif
 
+
+#if defined(USE_WIN32_CRYPTO) && \
+    !defined(LIBCURL_WINDOWS_OPENSSL_NEVER_USE_WINDOWS_CA_STORE)
```

Comment:
I don't think you need to do the `LIBCURL_WINDOWS_OPENSSL_NEVER_USE_WINDOWS_CA_STORE` thing. Asking for windows CA store should be explicit so disabling it in the build should not be necesary.